sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -992,6 +992,9 @@ case class FormatNumber(x: Expression, d: Expression)
 
       val sb = classOf[StringBuffer].getName
       val df = classOf[DecimalFormat].getName
+      val dfs = classOf[DecimalFormatSymbols].getName
+      val l = classOf[Locale].getName
+      val US = "US"
```

Comment:
I'll change it, it's no big deal. I think I'll also add a comment explaining this slightly non-intuivite approach of hard-coding a specific locale.